### PR TITLE
:sparkles: aws-account: manage the security contact

### PR DESCRIPTION
**Pull request by chassing**
Introduce the optional `securityContact` field to `/aws/account-1.yml`. 

Ticket: [APPSRE-10353](https://issues.redhat.com/browse/APPSRE-10353)

### CODE REVIEW

File: graphql-schemas/schema.yml [hunk 1]
```diff
@@ -1281,6 +1281,7 @@ confs:
   - { name: qontractReconcileVersion, type: string }
   - { name: terraformUsername, type: string }
   - { name: accountOwners, type: Owner_v1, isList: true, isRequired: true }
+  - { name: securityContact, type: AwsContact_v1 }
```
Comment (esron) [inline graphql-schemas/schema.yml:1284]: Nit: s/Aws/AWS/g

Comment (chassing) [reply graphql-schemas/schema.yml]: Fixed in last commit

File: schemas/aws/account-1.yml [hunk 2]
```diff
@@ -43,6 +43,24 @@ properties:
...
+        type: string
+        description: "The name of the security contact."
+      title:
+        type: string
+        description: "The title of the security contact. Default use the name."
+      email:
+        type: string
+        format: email
+      phoneNumber:
+        type: string
```
Comment (rporres) [inline schemas/aws/account-1.yml:59]: should we enforce something here, like `^+\d+` with a max length of 16? Just to make sure that we use proper international numbers here...

Comment (rporres) [reply schemas/aws/account-1.yml:59]: or at least what AWS accepts: https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-update-contact-alternate.html#manage-acct-update-contact-alternate-requirements

Comment (chassing) [reply schemas/aws/account-1.yml]: I added a very basic and simple phone number check. Should be enough for our use-case.

**Review by esron — APPROVED** [review]
LGTM

**Pull request merged**